charts/matrix-stack/configs/synapse/partial-haproxy.cfg.tpl

```diff
@@ -97,6 +97,16 @@ frontend synapse-http-in
   use_backend synapse-initial-synchrotron if is_sync !{ urlp("since") -m found } !{ urlp("from") -m found }
 {{- end }}
 
+{{- range $workerType, $workerDetails := (include "element-io.synapse.enabledWorkers" (dict "root" $root)) | fromJson }}
+{{- if include "element-io.synapse.process.hasHttp" (dict "root" $root "context" $workerType) }}
+{{- $workerTypeName := include "element-io.synapse.process.workerTypeName" (dict "root" $root "context" $workerType) }}
+{{- if include "element-io.synapse.process.canFallbackToMain" (dict "root" $root "context" $workerType) }}
+  acl {{ $workerType | replace "-" "_" }}_not_enough_capacity nb_srv(synapse-{{ $workerType }}) lt 1
```

Comment:
```suggestion
  acl {{ $workerType | replace "-" "_" }}_not_enough_capacity nbsrv(synapse-{{ $workerType }}) lt 1
```

Despite https://www.haproxy.com/blog/failover-and-worst-case-management-with-haproxy I couldn't find `nb_srv` in the HAProxy config reference. I could find [`nbsrv`](https://www.haproxy.com/documentation/haproxy-configuration-manual/latest/#7.3.1-nbsrv)